src/ui/components/DeckBuilder/DeckBuilder.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from '@storybook/react-vite'
+
+import { DeckBuilder } from './DeckBuilder'
+
+const meta: Meta<typeof DeckBuilder> = {
+  component: DeckBuilder,
+}
```

Comment:
This should include `title: 'Farmhand Shuffle/DeckBuilder'` to group it with the other stories. Also does this need the following to be consistent with the other stories like `CardQuantityControl`?

```js
parameters: {
  layout: 'centered',
},
tags: ['autodocs'],
```